gammapy/data/gti.py

```diff
@@ -172,14 +220,22 @@ def time_sum(self):
     @property
     def time_start(self):
         """GTI start times (`~astropy.time.Time`)."""
-        met = u.Quantity(self.table["START"].astype("float64"), "second")
-        return self.time_ref + met
+        return self.table["START"]
 
     @property
     def time_stop(self):
         """GTI end times (`~astropy.time.Time`)."""
-        met = u.Quantity(self.table["STOP"].astype("float64"), "second")
-        return self.time_ref + met
+        return self.table["STOP"]
+
+    @property
+    def met_start(self):
+        """GTI start time difference with reference time in sec, MET (`~astropy.units.Quantity`)."""
+        return (self.time_start - self.time_ref).to("s")
```

Comment:
Why go through `self.time_start` here if this is by definition what is stored in the table?